app/controllers/cms/content_controller.rb

```diff
@@ -22,12 +22,17 @@ class ContentController < Cms::ApplicationController
 
     # ----- Actions --------------------------------------------------------------
     def show
-      if @show_toolbar
-        render_editing_frame
-      else
-        render_page
+      begin
+        if @show_toolbar
+          render_editing_frame
+        else
+          render_page
+        end
+        cache_if_eligible
+      rescue => e
+        logger.debug "BrowserCMS Show action Exception: #{e.try(:message)}, backtace: #{e.backtrace}"
+        raise e
```

Comment:
Should be only `raise` and not `raise e`

Also we just could have done.
```
def show
  if @show_toolbar
    render_editing_frame
  else
    render_page
  end
  cache_if_eligible
rescue => e
  logger.debug "BrowserCMS Show action Exception: #{e.try(:message)}, backtace: #{e.backtrace}"
  raise
end
```

Also keep in mind you are using `debug` as the level. You need to make sure your log settings are in debug. I'd recommend changing it to info as opening for debug might print a lot of garbage you are not interested in

Reply:
@harvinder make sense Have updated the PR